webview/util.py

```diff
@@ -239,6 +240,10 @@ def js_bridge_call(window: Window, func_name: str, param: Any, value_id: str) ->
     def _call():
         try:
             result = func(*func_params)
+            if window.event_loop and asyncio.iscoroutine(result):
```

Comment:
We can probably throw an exception here. If the result is a coroutine, but the user hasn't configured the `event_loop` then we could throw an exception explaining they need to configure it.

Reply:
What benefit does an sync event handler bring here? Since a separate thread is launched for the handler anyway.

Reply:
The benefit is simply supporting async functions. There are plenty of reasons people might want to expose an existing async API to the bridge calls.

See the test file for usage.

Reply:
Alright fair enough.